Packs/CommonPlaybooks/Playbooks/playbook-Search_And_Delete_Emails_-_Generic_-_v2_6_0.yml

```diff
@@ -0,0 +1,1067 @@
+    type: regular
+    task:
+      id: b622809c-289a-413c-8019-3d5bafda95cc
+      version: -1
+      name: Search And Delete Emails - Agari Phishing Defense
+      description: Remediate suspected message.
+      script: Agari Phishing Defense|||apd-remediate-message
+      type: regular
+      iscommand: true
+      brand: Agari Phishing Defense
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      id:
+        complex:
+          root: incident
+          accessor: emailinternalmessageid
+      operation:
+        simple: delete
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1530,
+          "y": 1230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+system: true
+view: |-
+  {
+    "linkLabelsPosition": {
+      "4_13_Gmail": 0.81,
+      "4_15_Agari Phishing Defense": 0.86,
+      "4_2_#default#": 0.9
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1435,
+        "width": 3530,
+        "x": -1620,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: From
+  value:
+    complex:
+      root: incident
+      accessor: emailfrom
+  required: false
+  description: The value of the malicious email's "From" attribute.
+  playbookInputQuery:
+- key: Subject
+  value:
+    complex:
+      root: incident
+      accessor: emailsubject
+  required: false
+  description: The value of the malicious email's "Subject" attribute.
+  playbookInputQuery:
+- key: AttachmentName
+  value:
+    complex:
+      root: incident
+      accessor: attachmentname
+  required: false
+  description: The value of the malicious email's "AttachmentName" attribute.
+  playbookInputQuery:
+- key: SearchAndDeleteIntegration
+  value: {}
+  required: true
+  description: The integration in which to run the search and delete action. Can be
+    O365, or EWS.
+  playbookInputQuery:
+- key: O365ExchangeLocation
+  value:
+    complex:
+      root: incident
+      accessor: emailto
+  required: false
+  description: Used only in O365. Comma-separated list of mailboxes/distribution groups
+    to include, or use the value "All" to include all.
+  playbookInputQuery:
+- key: O365KQL
+  value: {}
+  required: false
+  description: Used only in O365. Text search string or a query that is formatted
+    using the Keyword Query Language (KQL).
+  playbookInputQuery:
+- key: O365Description
+  value: {}
+  required: false
+  description: Used only in O365. Description of the compliance search.
+  playbookInputQuery:
+- key: |
+    O365AllowNotFoundExchangeLocations
+  value:
+    simple: "false"
+  required: false
+  description: Used only in O365. Whether to include mailboxes other than regular
+    user mailboxes in the compliance search. Default is "false".
+  playbookInputQuery:
+- key: O365DeleteType
+  value:
+    simple: inputs.O365DeleteType
```

Comment:
How is an input got the value of the same input?

Reply:
legacy. fixed.